media/css/firefox/browsers/mobile.scss

```diff
@@ -149,6 +149,21 @@ $image-path: '/media/protocol/img';
     .header-product-ctas {
         display: none;
     }
+
+    // and also revert the grid to 1col
+    @media #{$mq-md} {
+        .c-page-header .mzp-c-split-container {
+            display: revert;
+            grid: revert;
+        }
+    }
+
+    // and avoid the noodles background where the form was
+    @media #{$mq-lg} {
+        .c-page-header .mzp-c-split-body {
+            width: 65%;
```

Comment:
**nit, non-blocking**: we might want to be a bit more conservative with avoiding the noodle background where "focus" pages are concerned because it's a darker background color. This was coming up for me on some landscape testing on tablets (around 1080px), minor edge cases

<img width="671" alt="body-overlap" src="https://github.com/user-attachments/assets/6083fc93-fb22-4695-a19d-3de316e90824">

Reply:
hang on, maybe this was an issue with browserstack, I'm not sure that background is supposed to be there given your screenshots

Reply:
I was actually considering removing the noodle graphics completely, if on mobile, thus in one column generally, yet on wide viewport. But that felt like too much design decisions to consider so I kinda cheated using 100% width only when there's no background, and bumped the 50% split to 65% when the background is present, to give the content a bit more space.

So this can:
- either wait for Craig's type changes to land, and update from that to see the impact,
- remove the part trying to cheat around the noodle background presence and keep it at 50%,
- or remove the noodle on mobile completely, no matter what screen size, and keep that in one column still,
- or get creative around applying css tricks/filters to the noodle to make it e.g. lighter(?) just for this specific case to be on the safe side,
- or add a solid white background to the paragraph text in case it runs into the image to keep legibility in worst case?

Reply:
> either wait for Craig's type changes to land, and update from that to see the impact,
remove the part trying to cheat around the noodle background presence and keep it at 50%,

these options sound best to me 

if the 50% doesn't seem like a generous enough line length on large screens, I would say the background removal is next best option (it's not really serving the purpose of highlighting a call to action anymore and readability takes priority)

Reply:
And of course, with the pending type scale changes it starts to move upwards and collide with the background more: _(so it's def not resilient enough, just running this over the other branch…)_

<img width="859" alt="Screenshot 2024-12-04 at 20 06 17" src="https://github.com/user-attachments/assets/e4e26c1f-0a63-4fe3-9108-8636226f746f">
<img width="846" alt="Screenshot 2024-12-04 at 20 05 44" src="https://github.com/user-attachments/assets/3a620f83-e73a-4c2b-87f0-e6cab2ba7435">

I'll experiment for a bit just out of curiosity, but eventually once these type changes land I'll rebase out most of it to just fill the empty space.

Reply:
So now with the type changes in prod I went ahead and did find some "worst case" viewports for this change:

<img width="1043" alt="Screenshot 2024-12-05 at 0 05 01" src="https://github.com/user-attachments/assets/741e307a-0475-42c9-9170-1930f6d671ff">
<img width="1050" alt="Screenshot 2024-12-05 at 0 04 20" src="https://github.com/user-attachments/assets/35208c52-299b-406c-81a4-64f86783d4e6">
<img width="992" alt="Screenshot 2024-12-04 at 23 50 04" src="https://github.com/user-attachments/assets/a3cf6a36-2816-464c-a19b-0543eaf73095">
<img width="989" alt="Screenshot 2024-12-04 at 23 48 02" src="https://github.com/user-attachments/assets/0fa06f42-7fa8-4a45-8175-bdb8799d2c4f">
<img width="1045" alt="Screenshot 2024-12-04 at 23 51 40" src="https://github.com/user-attachments/assets/7cf47452-337e-4466-8e27-33c75b7b9784">
<img width="1050" alt="Screenshot 2024-12-04 at 23 52 16" src="https://github.com/user-attachments/assets/ded14bce-7331-4780-97d8-a299dd526b5e">

(it's only hypothetical a landscape tablet would hit this exact resolution but not impossible…)

Reply:
So I went ahead and moved the noodle a few %% out of way, as it serves no purpose in this case without presenting the form, and the same viewport is no longer an issue:

<img width="1046" alt="Screenshot 2024-12-05 at 0 02 39" src="https://github.com/user-attachments/assets/14163861-7ba3-41d9-bb60-466619520e15">
<img width="1045" alt="Screenshot 2024-12-05 at 0 03 06" src="https://github.com/user-attachments/assets/926bee4a-8cc1-4476-9de6-ef1082cb638d">
<img width="1041" alt="Screenshot 2024-12-05 at 0 03 28" src="https://github.com/user-attachments/assets/e0797d63-e237-4099-90c9-775d056a4b2a">

A bit too specific perhaps, but tried;]

Reply:
There's no reason to cut this close. Give those noodles a real boot out of the way 🙂

Reply:
I think I'm happy now, only keeping a half of every illustration with some low effort styling. Will post some screens (tomorrow) to see if that's viable, or warrants more violent approach O:) — thanks for the feedback!